### DISCUSSION TITLE
hmac_secret - what? where?

### DISCUSSION THREAD

**Discussion by danielb2** [jeremyevans/rodauth#343] [General]
New to rodauth and reading the documentation at https://rodauth.jeremyevans.net/rdoc/files/README_rdoc.html which was the doc recommended as the first point of entry (start here if new).  I notice a lot of mentions of `hmac_secret` in this doc, but there's no indication of how to set it or no reference doc where configs are mentioned. 

> you are strongly encouraged to use the hmac_secret configuration method to set an HMAC secret

perhaps I'm missing it, but I wonder if there could be a link to how to actually set it from the doc there. Did I miss something obvious?

**Comment by jeremyevans** [comment]
`hmac_secret` is set like all other Rodauth settings.  It's one of the only settings explicitly listed in the example on the front page of the website: https://rodauth.jeremyevans.net/

Maybe it would be useful to copy the website example into the README?
